site/content/en/docs/Reference/API/Kustomization File/configMapGenerator.md

```diff
@@ -90,270 +90,3 @@ configMapGenerator:
   files:
   - myFileName.ini=whatever.ini
 ```
-
```

Comment:
How do we feel about moving the beginning of this file (same applies for `secretGenerator`) into Tasks as well?

We can totally do it later. This section just looks like it has some useful information, for example, the ability to specify a key with the `files` field and the example with the `options` field.

Reply:
Thanks for catching that. I added an `options` field as a new example to the `configMapGenerator` page.  I supplemented the "Create ConfigMap from a file" example with a smaller example demonstrating how to specify a key with the `files` field .